src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -159,32 +163,29 @@ public void serializeToView(ByteBuf buf)
         buf.writeInt(currentLevel);
         buf.writeInt(levels.size());
 
-        for(Level level : levels)
+        for (Level level : levels)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'level' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)